client-api/src/main/java/org/sonarsource/sonarlint/core/client/api/common/TelemetryData.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * SonarLint Core - Client API
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.client.api.common;
+
+public class TelemetryData {
+  private final long daysSinceInstallation;
+  private final long daysOfUse;
+  private final String version;
+  private final String product;
+  private final boolean connectedMode;
+
+  public TelemetryData(long daysSinceInstallation, long daysOfUse, String product, String version, boolean connectedMode) {
```

Comment:
The reason I didn't do it is to force setting all the values. We can add a builder and check if all fields are set later, if it grows.